service/rating.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require_relative 'file_handler'
+
+# Description/Explanation of Rating module
```

Comment:
You can delete this comment and skip following warnings in rubocop configuration.

Reply:
Fixed